pandas/tests/arithmetic/test_timedelta64.py

```diff
@@ -366,6 +367,8 @@ def test_subtraction_ops_with_tz(self):
         ts_tz2 = Timestamp("20130101").tz_localize("CET")
         dt_tz = ts_tz.to_pydatetime()
         td = Timedelta("1 days")
+        ts_utc = Timestamp("2020-10-22T22:00:00+00:00")
```

Comment:
can you update this test to use box_with_array fixture to ensure we test all the relevant types